docs/tutorials/init_cfg.md

```diff
@@ -0,0 +1,159 @@
+# Tutorial 10: Weight initialization
+
+During training, a proper initialization strategy is beneficial to speed up the training or obtain a higher performance. [MMCV](https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/utils/weight_init.py) provide some commonly used methods for initializing modules like `nn.Conv2d`. Model initialization in MMdetection mainly uses `init_cfg`. Users can initialize models with following two steps:
+
+1. Define `init_cfg` for a model or its components in `model_cfg`,  but `init_cfg` of children components have higher priority and will override `init_cfg` of parents modules.
+2. Build model as usual, but call `model.init_weights()` method explicitly, and model parameters will be initialized as configuration.
+
+The high-level workflow of initialization in MMdetection is :
+
+model_cfg(init_cfg) -> build_from_cfg -> model -> init_weight() -> initialize(self, self.init_cfg) -> children's init_weight()
+
+### init_cfg APIs
+it is dict or list[dict], and contains the following keys and values:
```

Comment:
Follow the markdonw format, should have an empty line between the title and the line.